saleor/graphql/schema.graphql

```diff
@@ -5043,19 +5043,33 @@ enum WeightUnitsEnum {
 
 type Wishlist implements Node {
   id: ID!
-  createdAt: DateTime!
-  items(before: String, after: String, first: Int, last: Int, id: ID): WishlistItemCountableConnection!
+  token: UUID!
+  user: User
+  items: [WishlistItem!]
+}
+
+type WishlistAddVariant {
+  errors: [Error!]!
+  wishlist: Wishlist
+  wishlistErrors: [WishlistError!]!
+}
+
+type WishlistRemoveVariant {
+  errors: [Error!]!
+  wishlist: Wishlist
+  wishlistErrors: [WishlistError!]!
 }
 
-type WishlistAddProductMutation {
+type WishlistUpdateVariant {
   errors: [Error!]!
-  wishlist: [WishlistItem]
+  wishlist: Wishlist
   wishlistErrors: [WishlistError!]!
 }
 
-type WishlistAddProductVariantMutation {
+type wishlistMoveItemsToCart {
   errors: [Error!]!
-  wishlist: [WishlistItem]
+  wishlist: Wishlist
+  lackOfQuantityVariants: [ID!]
```

Comment:
A bit more clear to me, but would like to hear some other opinions:
```suggestion
  outOfStockItems: [ID!]
```

Reply:
Yeah, that's better.